<launch>
  <node pkg="mpc_controller" type="mpc_controller" name="" output="screen">
      <param name="control/thresh" value="0.01"/>
      <param name="control/iter_max" value="200"/>
      <param name="control/car_len" value="0.3"/>
      <param name="control/traj_order" value="5"/>
      <param name="control/min_order" value="2.5"/>
      <param name="mpc/horizon_length" value="10" />
      <param name="mpc/longitudinal_state_cost" value="100.0" />
      <param name="mpc/lateral_state_cost" value="800.0" />
      <param name="mpc/orientation_state_cost" value="200.0" />
      <param name="mpc/terminal_state_cost" value="200.0" />
      <param name="mpc/linear_control_cost" value="25.0" />
      <param name="mpc/angular_control_cost" value="10.0"/>
      <!-- <param name="mpc/min_turning_radius" value="1.0" /> -->
      <param name="mpc/linear_velocity_max" value="1.0" />
      <param name="mpc/angular_velocity_max" value="1.0" />
      <remap from="odom" to="odom"/>
      <remap from="cmd_vel" to="cmd_vel"/>
      <remap from="traj2ctrl" to="traj2ctrl"/>
  </node> 
</launch>
